demos/general/demo_caiman_basic.py

```diff
@@ -137,27 +142,34 @@ def main():
 
     # %% visualize selected and rejected components
     cnm2.estimates.plot_contours(img=Cn, idx=cnm2.estimates.idx_components)
+
     # %% visualize selected components
     cnm2.estimates.view_components(images, idx=cnm2.estimates.idx_components, img=Cn)
+   
     #%% only select high quality components (destructive)
     # cnm2.estimates.select_components(use_object=True)
     # cnm2.estimates.plot_contours(img=Cn)
+
     #%% save results
     cnm2.estimates.Cn = Cn
     cnm2.save(cnm2.mmap_file[:-4]+'hdf5')
 
-# %% play movie with results (original, reconstructed, amplified residual)
-    cnm2.estimates.play_movie(images, magnification=4)
+    # %% play movie with results (original, reconstructed, amplified residual)
+    display_results = False
```

Comment:
Why this switch? I'm not sure it's much of a demo if it just runs and exits and the user sees nothing.

Reply:
This is the standard in all the notebooks where we suppress the opencv displays and people have to opt in. With this in particular it is run in demotests and I don't think we want opencv window popping up for tests (could be wrong we can change it back). There are other plots showing that use matplotlib.

Reply:
It makes sense for notebooks, but for CLI demos we're probably expecting them not to actively edit the code so much; a different default in my view makes sense for them because it never makes sense for a CLI demo to quietly run and do nothing visible by default.